src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -425,9 +467,6 @@ def downgrade():
         existing_type=postgresql.ARRAY(sa.VARCHAR()),
         nullable=True,
     )
-    op.drop_constraint(
-        "confighistory_asset", "privacyexperienceconfighistory", type_="foreignkey"
-    )
```

Comment:
Concept has been removed - we're not releasing with custom stylesheets. Missed schema migration cleanup here.